cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java

```diff
@@ -141,7 +141,7 @@
                                                        "   After Suite Function\n" +
                                                        "}\n" +
                                                        "@test:AfterSuite\n" +
-                                                       "function afterSuiteFunc () {\n" +
+                                                       "function afterSuiteServiceFunc () {\n" +
                                                        "    io:println(\"I'm the after suite function!\");\n" +
```

Comment:
Shall we update the println messages as well ?

Reply:
Updated message with 009fc6d